### PR TITLE
Update gnss.en.md - Info for GNSS Z value with Vertical grid shift in use

### PR DESCRIPTION
**Pull request by bettellam**
Info for GNSS Z value with Vertical grid shift in use

### CODE REVIEW

File: documentation/how-to/gnss.en.md [hunk 1]
```diff
@@ -267,6 +267,10 @@ Examples:
             is `NULL`. - `@position_source_name` > The value is
             `manual`.
 
+Info for GNSS Z value with Vertical grid shift in use:
+![image](https://github.com/opengisch/QField-docs/assets/23143342/2f0d6c78-7b38-47ab-bfd9-2e39d048cfa3)
```
Comment (suricactus) [inline documentation/how-to/gnss.en.md:271]: My five cents on this is if possible we store the images in our own repo.

Comment (suricactus) [reply documentation/how-to/gnss.en.md:271]: Cannot comment on the contents though, I guess @lucienicolier is a better reviewer.

Comment (SeqLaz) [reply documentation/how-to/gnss.en.md:271]: I moved the image to a our own repo

**Comment by nirvn** [comment]
Could we use an actual HTML table instead of an image of a table? It's much friendlier to consume, and makes it easy for our community to translate into other languages.

**Comment by bettellam** [comment]
@SeqLaz much nicer!

File: documentation/how-to/gnss.en.md [hunk 2]
```diff
@@ -267,6 +267,15 @@ Examples:
             is `NULL`. - `@position_source_name` > The value is
             `manual`.
 
+Info for GNSS Z value with Vertical grid shift in use:
```
Comment (nirvn) [inline documentation/how-to/gnss.en.md:270]: "Info" is a bit too informal here.

Comment (SeqLaz) [reply documentation/how-to/gnss.en.md:270]: Completely agree @nirvn

File: documentation/how-to/gnss.en.md [hunk 3]
```diff
@@ -267,6 +267,15 @@ Examples:
...
             `manual`.
 
+Information for GNSS Z value with Vertical grid shift in use:
+- *Antenna height compensation=False*
+
+| Vertical Grid Shift in use | point Z Value z(geometry) | GNSS Device z(@position_coordinate) | QField Display | QField Label                |
+|----------------------------|---------------------------|--------------------------------------|----------------|-----------------------------|
+| None                       | Z ellipsoidal device value| Z ellipsoidal device value           | Z ellipsoidal device value | Altitude: xxx.xxxx m       |
+| Orthometric from device    | Z orthometric device value| Z orthometric device value           | Z orthometric device value | Altitude: xxx.xxxx m (ortho.) |
+| USER_Shift_Grid.GTX        | Z shiftgrid value         | Z ellipsoidal device value           | Z shiftgrid value          | Altitude: xxx.xxxx m (grid) |
```
Comment (lucienicolier) [inline documentation/how-to/gnss.en.md:277]:
```suggestion
| USER_Shift_Grid.GTX <br> [vertical grid shift](#altitude-correction-/-vertical-grid-shift)        | Z shiftgrid value         | Z ellipsoidal device value           | Z shiftgrid value          | Altitude: xxx.xxxx m (grid) |
```

**Comment by bettellam** [comment]
@m-kuhn Do I have to do something?

**Comment by m-kuhn** [comment]
Ping @lucienicolier

**Review by suricactus — APPROVED** [review]

**Pull request merged**